internal/network/command_dns_forwarder_create.go

```diff
@@ -0,0 +1,93 @@
+package network
+
+import (
+	"github.com/spf13/cobra"
+
+	networkingdnsforwarderv1 "github.com/confluentinc/ccloud-sdk-go-v2-internal/networking-dnsforwarder/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *command) newDnsForwarderCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create [name]",
+		Short: "Create a DNS forwarder.",
+		Args:  cobra.MaximumNArgs(1),
+		RunE:  c.dnsForwarderCreate,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Create a DNS forwarder.",
+				Code: "confluent network dns forwarder create --domains abc.com,def.com --dns-server-ips 10.200.0.0,10.201.0.0 --gateway gw-123456",
+			},
+			examples.Example{
+				Text: "Create a named DNS forwarder.",
```

Comment:
```suggestion
				Text: `Create DNS forwarder named "my-dns-forwarder".`,
```

Reply:
I believe this wording was because the `display_name` field is optional in the API spec for all the networking `create` commands, the former is consistent with [the other networking commands](https://github.com/confluentinc/cli/pull/2451#discussion_r1408715092).